interfaces/payment-handler.idl

```diff
@@ -27,6 +28,21 @@ dictionary ImageObject {
              DOMString sizes;
              DOMString type;
 };
+[Constructor(DOMString type, CanMakePaymentEventInit eventInitDict),
```

Comment:
I was expecting to also see tests for the `CanMakePaymentEvent ` constructor and `CanMakePaymentEventInit`. I guess they are forthcoming?

Reply:
Yep. Adding tests step-by-step.

Reply:
Ok, cool. We can probably reuse quite a bit of: 
https://github.com/w3c/web-platform-tests/tree/master/payment-request/PaymentRequestUpdateEvent